src/figuresClasses.ts

```diff
@@ -1,19 +1,70 @@
+type Shape = 'triangle'| 'circle' | 'rectangle';
+type Color = 'red'| 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
```

Comment:
and use this enum here
```suggestion
  shape: Shape = Shape.Triangle;
```